frontend/src/app/modules/apiv3/endpoints/work_packages/api-v3-work-package-paths.ts

```diff
@@ -53,6 +53,9 @@ export class APIV3WorkPackagePaths extends CachableAPIV3Resource<WorkPackageReso
   // /api/v3/(?:projectPath)/work_packages/(:workPackageId)/available_projects
   public readonly available_projects = this.subResource('available_projects');
 
+  // /api/v3/(?:projectPath)/work_packages/(:workPackageId)/github_pull_requests
+  public readonly github_pull_requests = this.subResource('github_pull_requests');
```

Comment:
would be great to control this from a plugin

Reply:
it was said in slack, that this is actually fine ✅ 
it is more readable to not use more and more hooks, but directly write these things into the core